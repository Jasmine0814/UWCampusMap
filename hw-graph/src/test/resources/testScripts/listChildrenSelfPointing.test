# test if the children of a self-pointing can be listed.

CreateGraph A
AddNode A n1
AddEdge A n1 n1 e1
ListChildren A n1
