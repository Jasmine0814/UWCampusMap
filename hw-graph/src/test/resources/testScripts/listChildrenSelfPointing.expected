# test if the children of a self-pointing can be listed.

created graph A
added node n1 to A
added edge e1 from n1 to n1 in A

the children of n1 in A are: n1(e1)
